### ISSUE TITLE
The file name is not displayed when editing, if it is not in English

### ISSUE THREAD

**Issue by Ruslan-Aleev** [modxcms/revolution#14508]
### Summary
When editing a file, its name is not displayed, if it is not in English, see the picture below.

![russian_files](https://user-images.githubusercontent.com/12523676/55005178-f6a2f500-4ff4-11e9-8d69-b9ae50672aca.png)

Checked in Russian, at least.

### Environment
MODX 2.7.1
**Windows 7**

**Comment by JoshuaLuckers** [comment]
Do you get an error in the console?

**Comment by wax100** [comment]
![Screenshot_7](https://user-images.githubusercontent.com/4846064/55240158-424ddc80-5249-11e9-9b8d-c101d5013e39.jpg)
+1 , no errors in console

**Comment by Ruslan-Aleev** [comment]
No, there are no errors. You can create a file with a Russian name, for example, "Привет.txt" and try to check it yourself :)

**Comment by JoshuaLuckers** [comment]
I'm unable to reproduce this.

**Comment by Ruslan-Aleev** [comment]
@JoshuaLuckers Very strange, I tested it in Google Chrome and in Firefox - the bug repeats. OS Windows 7

**Comment by Jako** [comment]
It could be a charset/encoding issue in combination with json_encode.

**Comment by Electrica** [comment]
![image](https://user-images.githubusercontent.com/3878386/56601902-1a6e5080-661e-11e9-9222-66f1d7e9e827.png)
![image](https://user-images.githubusercontent.com/3878386/56601921-23f7b880-661e-11e9-8344-70b63d6f9372.png)
I don't have this bag.

**Comment by Ruslan-Aleev** [comment]
@Electrica Yes, in MODX3 there is no such bug.
But at the same time **in the title of the browser the file name in Russian is still not displayed**.

**Comment by JoshuaLuckers** [comment]
Is the locale set in system settings?

**Comment by Ruslan-Aleev** [comment]
@JoshuaLuckers The language settings for MODX 2.7.1 and MODX3 are the same, see the pictures below.

2:
![m2](https://user-images.githubusercontent.com/12523676/56638874-491d1300-6680-11e9-8a67-328bcef7aefa.png)

3:
![m3](https://user-images.githubusercontent.com/12523676/56638876-491d1300-6680-11e9-86de-74fcc8438459.png)

**Comment by JoshuaLuckers** [comment]
What happens when you set the locale?

**Comment by Ruslan-Aleev** [comment]
@JoshuaLuckers Yes it works! If set ru_RU.utf8 in locale.
In MODX 2.7.1. everything is correctly displayed.
And in MODX3 in the browser's titile the file name is correctly displayed.

**But the question remains**, why without a locale in MODX3, the names work, except for the browser titile, and in MODX 2.7.1 neither the names nor the title work?

**Comment by JoshuaLuckers** [comment]
Good question, what is the server configuration like? Windows, Linux etc?

**Comment by Ruslan-Aleev** [comment]
Both MODX 2.7.1 and MODX 3 are on the same server, here’s information about the server:

ОS: Ubuntu Server Edition
Apache: 2.4.10
Nginx: 1.13.4
MySQL: 5.7.21-20-beget
Perl: 5.26.1
PHP: 5.2, 5.3, 5.6, 7.0, 7.1, 7.2
Python: 2.7

**Comment by Jako** [comment]
What is the locale setting of the server/host? Maybe that uses ru_RU instead of ru_RU.utf8

MODX uses the locale setting of the host during init, if I remember right.

**Comment by JoshuaLuckers** [comment]
Related issue: #14500

**Comment by Jako** [comment]
Also related: #13079

**Comment by Ruslan-Aleev** [comment]
@Jako If put the ru_RU locale, then the errors are repeated.

**Comment by Jako** [comment]
@Ruslan-Aleev Please look into the discussion in #13079

The issue could not be fully solved.

**Comment by Jako** [comment]
Maybe your server/host uses a non utf8 locale. On all my installations I can't reproduce that issue in #13002. Maybe you get a way to solve that finally.

**Comment by Ruslan-Aleev** [comment]
@Jako I'm waiting for a response from the hosting technical support.

**Comment by Ruslan-Aleev** [comment]
@Jako Here is the support wrote: **$LANG = ru_RU.UTF-8**

**Comment by Ruslan-Aleev** [comment]
There are some problems with Windows and UTF-8 - https://en.wikipedia.org/wiki/Unicode_in_Microsoft_Windows#UTF-8

But I don’t understand why in MODX3 the error manifests itself differently.

**Comment by Jako** [comment]
The issue is caused by json_encode and a not utf8 capable locale. Because of this some server communication is decoded with issues or worse with a blank result.
